src/IECoreHoudini/bindings/IECoreHoudiniModule.cpp

```diff
@@ -106,8 +110,17 @@ static void *extractNodeFromHOM( PyObject *o )
 	return OPgetDirector()->findNode( homNode->path().c_str() );
 }
 
-#if UT_MAJOR_VERSION_INT >= 14
+#if UT_MAJOR_VERSION_INT >= 16
+// This little function returns the address of Houdini's shared QGLWidget
+// This can be necessary when wanting to create your own OpenGL context
+// which shares resources (textures, vertex buffers etc) with Houdini's contexts.
+// See GafferUI/GLWidget.py for an example of the hideous abuse this allows.
+static uint64_t sharedGLWidget()
+{
+	return (uint64_t)RE_Visual::getSharedGLWidget();	
```

Comment:
Did you check that this new thing has the same behaviour as the old `RE_QtVisual` equivalent?